src/scanner.cpp

```diff
@@ -5,6 +5,7 @@
  * (C) Cristina Cifuentes, Jeff Ledermann
  ****************************************************************************/
 
+#define NOMINMAX
```

Comment:
scanner.cpp and ast.cpp still have problem with std::min/max. VS tries to use non-std version.